src/navigation/appNavigation.js

```diff
@@ -385,6 +387,7 @@ const exchangeFlow = createStackNavigator({
 // SERVICES FLOW
 const servicesFlow = createStackNavigator({
   [SERVICES]: ServicesScreen,
+  [ADD_CASH]: AddCashScreen,
```

Comment:
Should probably be added to `AppFlowNavigation` as `servicesFlow` is to be removed soon

Reply:
Done